src/main/resources/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder/config.jelly

```diff
@@ -18,6 +18,14 @@
     <f:select style="width:100%" inline="true"/>
   </f:entry>
 
+  <f:entry title="Device Selection" field="deviceSelection" description="[Optional] Add the filters for device selection">
+    <f:textbox default="" />
+  </f:entry>
+
+  <f:entry title="Max Devices For Device Selection Filters" field="maxDevicesForDeviceSelection" description="[Optional] Add the max no of devices to choose that match the filters">
```

Comment:
https://docs.aws.amazon.com/devicefarm/latest/APIReference/API_DeviceSelectionConfiguration.html#devicefarm-Type-DeviceSelectionConfiguration-maxDevices

Isn't maxDevices required parameter?

Reply:
You are right. Seems like it is when you use deviceSelectionConfig but not when you use devicePool 
https://docs.aws.amazon.com/devicefarm/latest/APIReference/API_DevicePool.html 

I went with the thinking they both are the same. Will change this.